models/chainlink/arbitrum/chainlink_arbitrum_read_requests_feeds_daily.sql

```diff
@@ -0,0 +1,31 @@
+{{
+  config(
+    tags=['dunesql'],
+    alias=alias('read_requests_feeds_daily'),
+    partition_by=['date_month'],
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+    unique_key=['date_start', 'feed_address']
+  )
+}}
+
+{% set incremental_interval = '7' %}
+
+SELECT
+  'arbitrum' as blockchain,
+  MAX(CAST(date_trunc('month', date_start) as date)) as date_month,
+  CAST(date_trunc('day', date_start) AS date) AS date_start,
+  log_meta.feed_address,
+  MAX(feed_name) as feed_name,
+  COALESCE(COUNT(log_meta.feed_address), 0) as total_requests
+FROM {{ ref('chainlink_arbitrum_read_requests_feeds') }} log_meta
+{% if is_incremental() %}
+  WHERE
+   date_start >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
```

Comment:
We can use `incremental_predicate` to rewrite all the incremental filters:
i.e.
```suggestion
   {{ incremental_predicate('date_start') }}
```